assembly.md

```diff
@@ -0,0 +1,71 @@
+# Assembling the Horizontal WavetablePi
```

Comment:
Feel free to add your handle or name here like: `# Assembling the Horizontal WavetablePi (by sweetlilmre)`

Reply:
Something strange is happening, I added a commit and commented on this issue, but neither are pulling through

Reply:
Ah.. weird. It was a GH sync issue. I  seem to have fixed it but editing the PR and changing the base (well not really, just pointing it to your main branch again).